Shared/Supporting Files/Views/SampleList.swift

```diff
@@ -76,17 +86,29 @@ private extension SampleList {
                     VStack(alignment: .leading, spacing: 5) {
                         Text(sample.name)
                         if isShowingDescription {
-                            Text(sample.description)
-                                .font(.caption)
-                                .foregroundColor(.secondary)
-                                .transition(.move(edge: .top).combined(with: .opacity))
+                            if #available(iOS 16, *), deviceType != .phone {
+                                Text(sample.description)
+                                    .foregroundColor(selection == sample.name ? .white.opacity(0.8) : .secondary)
+                                    .font(.caption)
+                                    .transition(.move(edge: .top).combined(with: .opacity))
+                            } else {
+                                Text(sample.description)
+                                    .font(.caption)
+                                    .foregroundColor(.secondary)
+                                    .transition(.move(edge: .top).combined(with: .opacity))
+                            }
```

Comment:
There is a lot of repeated information in both branches. If this were just an availability check, just the necessary modifier(s) themselves could go in the branch. For example:

```swift
#if ... {
    .someModifier(...)
}
```

In this case, since there's a non-availability check as well, consider adding a computed property to determine the conditioned value:

```swift
var sampleForegroundColor: Color {
    ...
}
```

And then pass that to the `foregroundColor` modifier.

Remember: When it comes to branching (if, switch, etc.), a good rule of thumb is this: keep them as succinct as possible. If the same thing is in all branches, find a way to move it outside so that it only exists in one place.